pepper/test/e2e/trade_execution.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+source ./test/e2e/common.sh
+
+sed -i 's/#define ORDERS [0-9]*/#define ORDERS 3/g' apps/trade_execution.h
+sed -i 's/#define TOKENS [0-9]*/#define TOKENS 3/g' apps/trade_execution.h
+sed -i 's/#define ACCOUNTS [0-9]*/#define ACCOUNTS 1/g' apps/trade_execution.h
```

Comment:
Does having only 1 account in this scenario make this less general?

Reply:
It doesn't really matter from a high level point of view, since each token balance even if it belongs to the same account is separate. Moreover the current output format doesn't yet support accounts. We will update this once we get an example that supports more accounts.